### PR TITLE
[Data Mapper] Update APIs to allow JSON and large files

### PR DESCRIPTION
**Pull request by refortie**

### CODE REVIEW

File: apps/vs-code-data-mapper/src/DataMapperPanel.ts [hunk 1]
```diff
@@ -158,7 +158,10 @@ export default class DataMapperPanel {
     fs.readdir(path.join(DataMapperExt.getWorkspaceFolderFsPath(), schemasPath)).then((result) => {
       this.sendMsgToWebview({
         command: 'showAvailableSchemas',
-        data: result.filter((file) => path.extname(file).toLowerCase() === '.xsd'),
+        data: result.filter((file) => {
+          const fileExt = path.extname(file).toLowerCase();
+          return fileExt === '.xsd' || fileExt === '.json';
```
Comment (NicL9923) [inline apps/vs-code-data-mapper/src/DataMapperPanel.ts:163]: Should this be hooked up to supportedSchemaFileExts too?

File: libs/data-mapper/src/lib/components/inputDropdown/InputDropdown.tsx [hunk 2]
```diff
@@ -39,7 +39,7 @@ const customValueDebounceDelay = 300;
 interface SharedOptionData {
```
Comment (NicL9923) [inline libs/data-mapper/src/lib/components/inputDropdown/InputDropdown.tsx:39]: I think you may need to add the .json file extension to the file browse button wherever it is in the ConfigPanel stuff as well if you haven't already

**Review by NicL9923 — APPROVED** [review]

File: apps/vs-code-data-mapper/src/DataMapperPanel.ts (hunk 1 above)
Comment (refortie) [reply apps/vs-code-data-mapper/src/DataMapperPanel.ts:163]: Changed

File: libs/data-mapper/src/lib/components/inputDropdown/InputDropdown.tsx (hunk 2 above)
Comment (refortie) [reply libs/data-mapper/src/lib/components/inputDropdown/InputDropdown.tsx:39]: https://github.com/Azure/LogicAppsUX/pull/1755/files#diff-066ad794c957583c3272618fe1455d04ee2c3927884bbe93eb17ce238f7520afR103 This?

**Pull request merged**